### DISCUSSION TITLE
How do i build only meshlab screened poisson plugin

### DISCUSSION THREAD

**Discussion by sree9578** [cnr-isti-vclab/meshlab#1378] [Q&A]
I am trying to build meshlab screened poisson plugin , but i am facing a lot of errors . I planning to run the plugin as command line tool , can the team help me on this ?
I am also interested on how the mesh expands over the rectangular area , if you can explain that , it would be of great help.

**Comment by cignoni** [comment]
if you need it as a command line tool just use pymeshlab

**Reply by sree9578** [reply]
I am looking for developing in c++ , setting aside python is there a way to build them as separate executable which i can use it ?

Also any information over on how the mesh expands over the rectangular area using Poisson Reconn , as i checked the Poisson reconn from mkazhdan professor with same parameter ,as it is generating me a different mesh over only where the points are there , but here it is widely extended over a rectangular area like bbox
